apps/web/components/EditablePage/EditablePage.vue

```diff
@@ -15,21 +15,23 @@
       @end="handleDragEnd"
     >
       <template #item="{ element: block, index }">
-        <PageBlock
-          :index="index"
-          :block="block"
-          :disable-actions="disableActions"
-          :is-clicked="isClicked"
-          :clicked-block-index="clickedBlockIndex"
-          :is-tablet="isTablet"
-          :block-has-data="blockHasData"
-          :change-block-position="changeBlockPosition"
-          :root="getBlockDepth(block.meta.uuid) === 0"
-          class="group"
-          :class="getBlockClass(block).value"
-          data-testid="block-wrapper"
-          @click="tabletEdit(index)"
-        />
+        <component :is="block?.meta?.narrowContainer ? NarrowContainer : 'div'">
```

Comment:
I'm still not sure this is the way. It works if you have a single `CategoryData` block at the top of the page, but if you put the category data somewhere else, or have it multiple times, I suspect this would look pretty bad. Doesn't `CategoryData` itself have padding options that can accomplish the same thing?

Reply:
If it's a prospective setting, shouldn't it be `config.layout.narrowContainer` then?